### DISCUSSION TITLE
I haven't received education benefits yet

### DISCUSSION THREAD

**Discussion by 544r** [community/community#143433] [GitHub Education]
### Hi everyone,

<img width="245" alt="屏幕截图 2024-11-03 165057" src="https://github.com/user-attachments/assets/072452df-3f18-4d1a-8067-fb2f0dff87c0">
As shown in the picture, 2 days have passed, but i have not received education benefits

**Comment by aminefjr** [comment]
Hi @544r ,

I think you have to wait a week

**Comment by davevad93** [comment]
Hi @544r ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @544r was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
